src/pages/Profile.tsx

```diff
@@ -1,6 +1,30 @@
+import { useState, useEffect } from 'react';
+import { getUserData } from '../services/user.api';
+import { IUser } from '../types/types';
+
 import icon from '../assets/icon_pizza_shop.jpg';
 
 const Profile = () => {
+  const [userData, setUserData] = useState<IUser>({
+    name: '',
+    direction: '',
+    email: '',
+    telephone: 0,
+  });
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const result = await getUserData();
+        setUserData(result as IUser);
```

Comment:
Avoid casting.  getUserData should return the type/interface IUser.
Unknown was used because there was no IUser data type/interface.
The interface also should be in a separate file. IUser data will be used in other places.

Reply:
Done